gui/wxpython/gis_set.py

```diff
@@ -542,17 +542,19 @@ def OnCreateLocation(self, event):
             create_location_interactively(self, self.gisdbase)
         )
         if location is not None:
+            if user_mapset != "PERNAMENT":
```

Comment:
If there is really a different treatment of PERMANENT mapset needed, it needs to be documented here.

Reply:
It is not a different treatment.

Reply:
I don't understand. Are you saying that PERMANENT and any other mapset is treated the same here? I still see the same condition `if mapset != "PERMANENT":` in the code.

Reply:
This condition is not needed here because when a location is not None, a user mapset is also always different from None. And I need to treat all mapsets the same way.